build.gradle

```diff
@@ -1,26 +1,43 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-
 buildscript {
     apply from: "gradle/dependencies.gradle"
     repositories {
         google()
         jcenter()
+        mavenCentral()
+        maven {
+            url 'https://api.mapbox.com/downloads/v2/releases/maven'
+            authentication {
+                basic(BasicAuthentication)
+            }
+            credentials {
+                username = "mapbox"
+                password = System.getenv("SDK_REGISTRY_TOKEN") ?: project.property("SDK_REGISTRY_TOKEN") as String
```

Comment:
Could you add installation instructions to readme? There's already a small section about the token that will have to get refactored now.

Reply:
Will do